src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.cs

```diff
@@ -20,6 +20,19 @@ namespace Microsoft.CodeQuality.Analyzers.Maintainability.UnitTests
     public class ReviewUnusedParametersTests
     {
         #region Unit tests for no analyzer diagnostic
+        [Fact]
+        [WorkItem(4039, "https://github.com/dotnet/roslyn-analyzers/issues/4039")]
+        public async Task NoDiagnosticForUnnamedParameterTest()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+public class NeatCode
+{
+    public void DoSomething(string)
+    {
+    }
+}
+", DiagnosticResult.CompilerError("CS1001").WithLocation(4, 35));
+        }
```

Comment:
This only has a C# test. I think there is no parallel VB test. I can't find any way to get an empty parameter name in VB syntax tree. For example:

```
Public Sub M(As String)
```

This generates "As" as the name, and "String" as skipped trivia.